SabreTools/Features/BaseFeature.cs

```diff
@@ -1705,6 +1705,20 @@ internal static Feature RootDirStringInput
             }
         }
 
+        internal const string SymlinkDirStringValue = "symlink-dir";
+        internal static Feature SymlinkStringInput
+        {
+            get
+            {
+                return new Feature(
+                    SymlinkDirStringValue,
+                    new List<string>() { "-sd", "--symlink-dir" },
```

Comment:
I'm pretty sure that `-sd` is already a valid flag for SuperDAT. This needs to be replaced.